javascript/packages/orchestrator/src/providers/k8s/kubeClient.ts

```diff
@@ -960,6 +973,7 @@ export class KubeClient extends Client {
 
   async isPodMonitorAvailable() {
     let available = false;
+    if (process.env.ZOMBIE_PODMONITOR === "0") return available;
```

Comment:
We can assume that if is running in _ci_ we always have podmonitor right?

Reply:
I just wanted them to disable it for now for us for the migration period and not to break it for other users